src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -58,6 +58,20 @@ public DebugService(PowerShellContext powerShellContext)
         #region Public Methods
 
         /// <summary>
+        /// Disables all breakpoints in the runspace.
+        /// </summary>
+        /// <returns></returns>
+        public async Task DisableAllBreakpoints()
```

Comment:
Nevermind. I'm an idiot. Every time I re-install the alpha I have to remember to update the package.json package for the PS extension to use my debug bits.  Forgot to do that. :-(